docs/runtime/shell.md

```diff
@@ -406,7 +405,8 @@ For simple shell scripts, instead of `/bin/sh`, you can use Bun Shell to run she
 
 To do so, just run the script with `bun` on a file with the `.bun.sh` extension.
 
-```sh#script.bun.sh
```

Comment:
@Jarred-Sumner I mean you obviously know better, but this is not even visible on github

Reply:
okay, i see, this is docs for website, and github is secondary